mayastor/src/replicas/rebuild/rebuild_impl.rs

```diff
@@ -54,6 +56,22 @@ pub(super) struct RebuildTasks {
     segments_done: u64,
 }
 
+/// Checks whether a range is contained within another range
+pub trait Within<T> {
+    /// True if `right` is contained within `self`, otherwise false
```

Comment:
This comment differs from the implementation on lines 68-71. It checks if "self" is within "right".

Reply:
Done